assets/chat/js/chat.js

```diff
@@ -1760,6 +1760,8 @@ class Chat {
           ).join(', ')}.`
         ).into(this);
       }
+    } else if (parts.some((username) => username === this.user.nick)) {
```

Comment:
It's possible the username passed into the command doesn't match the case of the user's username, resulting in false negatives. You should normalize the usernames before comparing.

Reply:
Good catch - normalized casing of both sides in latest commit.